### PR TITLE
 [Localization] Traditional Chinese (zh_TW) translations for all dialogues

### PR DESCRIPTION
**Pull request by Mewtwo2387**
## What are the changes?
Translated all of the Traditional Chinese (zh_TW) dialogues. 
Most of these translations are modified from the Simplified Chinese (zh_CN) translations in #1903 or [the Simplified Chinese paratranz project](https://paratranz.cn/projects/10462)

Similar to #1903, eslint fails as it requires double quotations marks, which cannot be used here as we need to use $ to split sentences. 

## Why am I doing these changes?
Only Simplified Chinese dialogues are included in #1903 and I have confirmed that they are only going to work on Simplified Chinese. So, I took those translations and translated it into Traditional Chinese too.

## What did change?
`src/locales/zh_TW/dialogue.ts` only. 

## Screenshots/Videos
image

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/zh_TW/dialogue.ts [hunk 1]
```diff
@@ -4,2232 +4,2069 @@ import {DialogueTranslationEntries, SimpleTranslationEntries} from "#app/plugins
...
     },
     "victory": {
-      1: "I certainly found an interesting Trainer to face!"
+      1: `我發現了一個有趣的訓練師對手！`,
     },
     "defeat": {
-      1: "Ahaha. What an interesting battle."
-    }
-  },
-  "rika": {
```
Comment (Tempo-anon) [inline src/locales/zh_TW/dialogue.ts:1206]: rika is missing?

Comment (40chyan) [reply src/locales/zh_TW/dialogue.ts:1206]: `  "rika": {
    "encounter": {
      1: `我要对你手下留情，但……骗你的啦！$好好动脑！`,
    },
    "victory": {
      1: `不错，小子。`,
    },
    "defeat": {
      1: `啊哈哈哈哈！你真的很特别，小子！`,
    },
  },`
  
  think i might missed it just like hala, you can transfer it into zh_TW version and add it.

**Comment by Mewtwo2387** [comment]
Got some issues with git so I rebased and force pushed, rika's quotes should be there now

**Comment by 40chyan** [comment]
@Mewtwo2387 zh_cn dialogue has been merged, you can copy and transfer that file into zh_tw.

**Comment by Mewtwo2387** [comment]
Closing this for now, will reopen an updated one later

**Pull request closed without merging**